doc/reference/common-weaknesses/double-satisfaction.rst

```diff
@@ -0,0 +1,73 @@
+Double satisfaction
+===================
+
+Suppose we have a validator V that implements a typical “atomic swap” or “escrowed swap” between A and B where A goes first, i.e. V says:
+
+    This output can only be spent if, in the same transaction, there is an output sending the agreed-upon payment (encoded in the output’s datum) to A.
+
+Now suppose that A and B have two swaps in progress, one for a token T1 at the price of 10 Ada, and one for a token T2 at the same price.
```

Comment:
Do you mean A create two outputs which sell token T1 at the price of 10 Ada, and sell token T2 at the same price?